src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -460,6 +460,34 @@ def test_containerapp_env_internal_only_e2e(self, resource_group):
         ])
 
 
+    @ResourceGroupPreparer(location="eastus")
+    @live_only()  # passes live but hits CannotOverwriteExistingCassetteException when run from recording
```

Comment:
Please remove `live_only`.
What cause `CannotOverwriteExistingCassetteException` when run from recording?

Reply:
We do not recommend add `live_only` to test, if it is required, please find out what cause the problem and add more comments in test case.

Reply:
I saw the same thing occurring with `test_containerapp_env_internal_only_e2e`. @bgashirabake offered an explanation here: https://github.com/Azure/azure-cli-extensions/pull/6736#discussion_r1320353783

Reply:
Will be creating a WI to remove all live_only and resolve exception.

Reply:
For previous tests, we often copy paste from old ones, which causes many test cases are added with live_only() but they do not have issue to replay. All test cased with live_only will be skipped in CI test. Soif possible, we should try to not add live_only for tests which could help detect unepected changes to your scenarios introduced by other PRs.

Reply:
please try to remove this @live_only and run the test in live mode.